easybuild/easyblocks/b/boost.py

```diff
@@ -120,6 +123,13 @@ def prepare_step(self, *args, **kwargs):
     def configure_step(self):
         """Configure Boost build using custom tools"""
 
+        # boost_multi_thread is deprecated
+        if self.cfg['boost_multi_thread'] is not None:
+            self.log.deprecated("boost_multi_thread has been deprecated, it has been replaced by tagged_layout. "
```

Comment:
@akesandgren The easyconfigs for Boost 1.74.0 should be updated accordingly, since they set `boost_multi_thread = True`.

Couldn't find a PR for that, did I overlook it?

Reply:
No I didn't make one yet. Didn't know if we wanted to change that already or not.